docs/pages/access-controls/access-request-plugins/notification-routing-rules.mdx

```diff
@@ -0,0 +1,110 @@
+- Using the Web UI dialog.
+- Creating a YAML resource file containing the definition of a rule.
+
+### Create an Access Monitoring Rule via the Web UI
+
+To create an Access Monitoring Rule via the Web UI, first navigate to
+the Access Request page and click on `View Notification Routing Rules`.
+
+![View Notification Routing Rules](../../../img/enterprise/plugins/access_monitoring_rules/view_routing_rules.png)
+
+Then click `Create Notification Rule`.
+
+![Create AMR](../../../img/enterprise/plugins/access_monitoring_rules/access_monitoring_rule_ui.png)
+
+If a plugin that supports Access Monitoring Rule based routing is not enrolled
+the UI will prompt you to enroll one.
+
+### Create an Access Monitoring Rule using YAML resource
+
+Here is an example Access Monitoring Rule. This rule will notify `your_slack_channel`
+via the Slack plugin if an Access Request containing the role `your_role_name` is
+made.
+
+```yaml
+kind: access_monitoring_rule
+version: v1
+metadata:
+  name: your-plugin-name
+spec:
+  subjects: ['access_request']
+  condition: 'access_request.spec.roles.contains("your_role_name")'
+  notification:
+    name: 'slack'
+    recipients: ['your_slack_channel']
+```
+
+The above routing rule can be created using `tctl create -f your-file-name.yaml`
+from the command line.
+
+Multiple recipients can be specified in an Access Monitoring Rule.
+The condition field of the rule is set to a predicate expression defining the conditions
+under which you wish the rule to apply.
+
+Fields of the Access Request that are currently supported are
+| Field | Description |
+| --- | --- |
+| access_request.spec.roles | The set of roles requested. |
+| access_request.spec.suggested_reviewers | The set of reviewers specified in the request. |
+| access_request.spec.system_annotations | A map of system annotations on the request. |
+| access_request.spec.user | The requesting user. |
+| access_request.spec.request_reason | The request reason. |
+| access_request.spec.creation_time | The creation time of the request. |
+| access_request.spec.expiry | The expiry time of the request. |
+
+Predicate expressions used in the condition of Access Monitoring Rules must evaluate to
+either true or false.
+
+Some example values for the condition field and their possible uses.
+- `access_request.spec.user == "some_user"` - Routing based on requesting user.
+- `access_request.spec.roles.contains("some_role")` - Routing based on the requested roles.
+- `access_request.spec.roles.contains_any(set("role_1", "role_2"))` - Routing based on multiple roles.
+
+## Step 2/2. Testing your new Access Monitoring Rule
+
+In the above example rule for the Slack plugin.
+- Replace the role "your_role_name" with a role of your choice.
+- Replace "your_slack_channel" with a slack channel your plugin has access to.
+
+Afterwards create an Access Request for the role you specified in the rule.
+
+(!docs/pages/includes/plugins/create-request.mdx!)
```

Comment:
We haven't described the expected outcome.

In order to test that the rule works, you need to
- create the rule
- create a matching access request
- verify that the slack notification happened (this step is missing)